packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -622,7 +638,7 @@ async function renameFile(file, entry) {
   }
   log('debug', `Renaming ${getAttribute(file, 'name')} to ${entry.filename}`)
   try {
-    await cozy.files.updateAttributesById(file._id, { name: entry.filename })
+    await client.collection(FILES).updateAttributes(file._id, { name: entry.filename })
```

Comment:
You should use client.save({...file, name: entry.filename}). 

It will call updateAttributes by itself: https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/FileCollection.js#L513-L524


You should avoid the call to client.collection as much as possible. As soon that a method is available through the cozy-client DSL we should use it!